extensions/tokenizers/rust/src/models/mod.rs

```diff
@@ -234,3 +234,21 @@ pub extern "system" fn Java_ai_djl_engine_rust_RustLibrary_runInference<'local>(
         }
     }
 }
+
+pub fn as_device(device: &String) -> Result<Device> {
```

Comment:
Can we re-use existing `as_device()`

Reply:
Ok, I have modified the existing `as_device()` a little bit, please review.